azurerm/internal/services/subscription/data_source_subscriptions_test.go

```diff
@@ -17,6 +17,7 @@ func TestAccDataSourceSubscriptions_basic(t *testing.T) {
 		{
 			Config: SubscriptionsDataSource{}.basic(),
 			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("subscriptions.0.resource_manager_id").Exists(),
```

Comment:
can we make this just `id` to match other resources:

```suggestion
				check.That(data.ResourceName).Key("subscriptions.0.id").Exists(),
```